Scripts/Services/Pet Training/SpecialAbility.cs

```diff
@@ -1454,8 +1454,8 @@ protected override void OnTick()
                 }
                 else
                 {
-                    var dam = AOS.Damage(Defender, Attacker, 5, 0, 0, 0, 0, 100);
-                    Attacker.Hits = Math.Min(Attacker.HitsMax, Attacker.Hits + dam);
+                    var dam = AOS.Damage(Defender, Attacker, 2, 0, 0, 0, 0, 100);
```

Comment:
The link you posted a couple days ago says the initial damage is 45 and each damage tick thereafter is 5. You said you tested on EA, but were you taking into account that it was energy damage and the targets energy resists?

Reply:
I'm testing with the naked character.